service/storage_service.cc

```diff
@@ -436,7 +436,7 @@ future<> storage_service::topology_transition(storage_proxy& proxy, cdc::generat
         }
     } catch (replica::no_such_column_family& e) {
         slogger.error("Error while applying topology mutations from {}: {}", from, e);
-        throw std::runtime_error(fmt::format("Error while applying topology mutations: {}", e));
+        throw std::runtime_error(format("Error while applying topology mutations: {}", e));
```

Comment:
this reminds me that @avikivity also fixed a similar issue in Seastar.  see https://github.com/scylladb/seastar/pull/1249 . that issue only exhibits itself on aarch64.

Reply:
i need to take a closer look at this backtrace pasted by Avi.